ppr-ui/src/components/parties/SecuredParties.vue

```diff
@@ -647,4 +691,16 @@ export default defineComponent({
 .smaller-button {
   padding: 0 12px !important;
 }
+
+.actions-width {
+  width: 140px;
+}
+
+.actions-border {
+  border-left: 1px solid #f1f3f5;
```

Comment:
Can you use the theme for the colour here please?

Reply:
done